docs/src/main/paradox/user/lang/index.md

```diff
@@ -1,5 +1,6 @@
 # Languages
 
+## Officially supported client libraries
```

Comment:
I'm not sure we should divide things into Officially supported and not officially supported—instead I think we should list "Known language supports" and then list supported features and versions.

Reply:
Guess I've been working for editors for too long 😄 
What about removing the `Officially supported client libraries` title and renaming the other one to `3rd party libraries`?

Reply:
Sounds like a great idea, @ChristopherAndre! Make that change and then I think we are ready to merge

Reply:
Done!